### PR TITLE
linphone: 4.2.4 -> 4.2.5 and update all major SIP-related dependencies

### PR DESCRIPTION
**Pull request by schmittlauch**
###### Motivation for this change

This fixes a crash on incoming calls for me.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by schmittlauch** [comment]
just a small commit message fix

**Comment by schmittlauch** [comment]
@GrahamcOfBorg build linphone

File: pkgs/development/libraries/bzrtp/default.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation rec {
   pname = "bzrtp";
-  version = "4.4.0";
+  version = "4.4.9";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/bzrtp/default.nix:10]: I think your forgot to change the hash here.

Comment (schmittlauch) [reply pkgs/development/libraries/bzrtp/default.nix:10]: Don't ask me why, but `1yjmsbqmymzl4r7sba6w4a2yld8m6hzafr6jf7sj0syhwpnc3zv6` is indeed the correct hash for bzrtp-4.4.9, I tried changing it but that is what got fetched.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>twinkle</li>
  </ul>
</details>
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>bcg729</li>
    <li>bctoolbox</li>
    <li>belcard</li>
    <li>belle-sip</li>
    <li>belr</li>
    <li>bzrtp</li>
    <li>liblinphone</li>
    <li>lime</li>
    <li>linphone</li>
    <li>mediastreamer</li>
    <li>mediastreamer-openh264</li>
    <li>msilbc</li>
    <li>ortp</li>
    <li>trx</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Also a bunch of

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

`lgpl21` is a deprecated license, check if project uses `lgpl21Plus` or `lgpl21Only` and change `meta.license` accordingly.

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

**Comment by schmittlauch** [comment]
I specified the exact license variants, anything else I need to fix?

File: pkgs/development/libraries/bzrtp/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/libraries/bzrtp/default.nix:10]: thats weird.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>twinkle</li>
  </ul>
</details>
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>bcg729</li>
    <li>bctoolbox</li>
    <li>belcard</li>
    <li>belle-sip</li>
    <li>belr</li>
    <li>bzrtp</li>
    <li>liblinphone</li>
    <li>lime</li>
    <li>linphone</li>
    <li>mediastreamer</li>
    <li>mediastreamer-openh264</li>
    <li>msilbc</li>
    <li>ortp</li>
    <li>trx</li>
  </ul>
</details>

**Pull request merged**